pkg/util/bundle/extract.go

```diff
@@ -27,12 +28,15 @@ func Extract(ctx *pulumi.Context, bundleURL, shasumURL string) (*local.Command,
 	if err != nil {
 		return nil, err
 	}
+	if err := os.Chmod(*scriptfileName, 0777); err != nil {
+		return nil, err
+	}
 	execScriptENVS := map[string]string{
 		"BUNDLE_DOWNLOAD_URL":     bundleURL,
 		"SHASUMFILE_DOWNLOAD_URL": shasumURL}
 	return local.NewCommand(ctx, "execExtractScript",
 		&local.CommandArgs{
-			Create:      pulumi.String(fmt.Sprintf(". %s", *scriptfileName)),
+			Create:      pulumi.String(fmt.Sprintf("%s", *scriptfileName)),
```

Comment:
Is it picking the Envs? being executed like this...wihtout `. script`??

Reply:
yes, it's using the env we are passing as part of the local.Command